nginx/dev/nginx.conf

```diff
@@ -0,0 +1,39 @@
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+	worker_connections 1024;
+}
+
+http {
+	server {
+		listen 80;
+		server_name localhost 127.0.0.1;
+
+		location / {
+			proxy_pass	http://frontend:3000;
+
+			proxy_set_header Host $host;
+			proxy_set_header	X-Real-IP $remote_addr;
+			proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
+			proxy_set_header   X-Forwarded-Host $server_name;
+
+			proxy_redirect off;
+
+			proxy_http_version 1.1;
+			proxy_set_header Upgrade $http_upgrade;
+			proxy_set_header Connection "upgrade";
+		}
+
+		location /api {
```

Comment:
You may need to set some sane defaults for CORS https://enable-cors.org/server_nginx.html

Reply:
Will we be needing this if we host both the services under the same domain?